homeassistant/components/zha/strings.json

```diff
@@ -29,6 +29,19 @@
       "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
     }
   },
+  "config_panel": {
```

Comment:
I don't think that this makes sense as long as the actual panel code lives in the frontend. Or are these strings passed to the frontend from the backend?

CC @bramkragten

Reply:
They use `ha-form` and get the actual data/fields from the backend, so I think it makes sense to have the translations on the backend.

Reply:
@balloob are we good with this now?

Reply:
Yes